packages/neuron-ui/src/widgets/Table/index.tsx

```diff
@@ -54,15 +54,17 @@ export default ({
         ))}
       </tbody>
     </BsTable>
-    <Row>
-      <Col>
-        <Pagination
-          currentPage={pageNo}
-          pageSize={pageSize}
-          total={totalCount}
-          onChange={page => onPageChange(page, pageSize)}
-        />
-      </Col>
-    </Row>
+    {totalCount <= pageSize ? null : (
```

Comment:
Does `totalCount > pageSize && (...)` work as this?

Reply:
It does, but the expression returns `false`, and the ternary returns `null`, they are slightly different in semantics.

Reply:
1. `A && Component` returns A if A is false-like, which has side-effects in some cases.
E.g. `{items.length && <Items />}`, `Items` is expected to be hidden when `items.length === 0`, but `0`, which is false-like, is rendered.

2. `A ? Component : null` is under controller to show `nothing` or `Component`